bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -103,6 +105,7 @@ Property,Water,,Water,,
 Property,Wind,,Wind,,
 Equipment,AlarmDevice,,Alarm Device,,
 Equipment,AlarmSystem,,Alarm System,Alarm Systems,
+Equipment,Application,,Application,"App, Software program",
```

Comment:
```suggestion
Equipment,Application,,Application,Software program,
```
The synonym mustn't match another existing tag. All synonyms from all tags must be unique to avoid ambiguities.